src/components/pages/order/Main/Admin/TabsConfig.js

```diff
@@ -0,0 +1,18 @@
+import { AiOutlinePlus } from "react-icons/ai";
+import { MdModeEditOutline } from "react-icons/md";
+
+export const tabsConfig = [
+  {
+    index: "edit",
+    label: "Modifier un produit",
+    icon: <MdModeEditOutline className="icon" />,
```

Comment:
```suggestion
    icon: <MdModeEditOutline />,
```

"className" ne doit pas être ici dans un fichier de config et non de JSX. Faut déplacer className dans le fichier de composant / JSX. Ici, limite un non techos est censé modifier ce fichier sans répercussion sur le code.